pallets/gear/rpc/runtime-api/src/lib.rs

```diff
@@ -22,8 +22,11 @@ use codec::Codec;
 use sp_std::vec::Vec;
 
 sp_api::decl_runtime_apis! {
-    pub trait GearApi<ProgramId>
-    where ProgramId: Codec {
-        fn get_gas_spent(program_id: ProgramId, payload: Vec<u8>) -> Option<u64>;
+    pub trait GearApi<AccountId, ProgramId>
+    where
+    AccountId: Codec,
+    ProgramId: Codec,
+    {
+        fn get_gas_spent(account_id: AccountId, program_id: ProgramId, payload: Vec<u8>, kind: Vec<u8>) -> Option<u64>;
```

Comment:
why kind is not enum? 

also how do you specify message you are replying to?

Reply:
`Vec` (or something similar to string) is simpler to send (no need to encode as the JS side may not have info about enum).

Regarding message ID, should we add the correspondent parameter or create a dedicated RPC call? We are to provide `ExitCode` too for more accurate calculation.

Reply:
It should be strongly typed, no `Vec` of course

of course you should calculate by creating probe reply to particular message id, and also provide exitcode, otherwise it's just nonsense

Reply:
Fixed.